src/mysqlsh_helpers.py

```diff
@@ -46,15 +71,34 @@ def mysqlsh_common_dir(self) -> str:
         Returns:
             Path to common dir
         """
-        return "/root/snap/mysql-shell/common"
+        if os.path.exists("/root/snap/mysql-shell/common"):
+            return "/root/snap/mysql-shell/common"
+        return "/tmp"
 
     def configure_mysql_users(self):
         """Configure the MySQL users for the instance.
 
-        Creates a 'clusteradmin' user with the appropriate privileges and
-        revokes certain privileges from the 'root' user.
+        Creates base `root@%` and `clusteradmin@instance_address` user with the
+        appropriate privileges, and reconfigure `root@localhost` user password.
         """
-        pass
+        _script = (
+            "SET @@SESSION.SQL_LOG_BIN=0;",
+            f"CREATE USER '{self.cluster_admin_user}'@'{self.instance_address}' IDENTIFIED BY '{self.cluster_admin_password}';",
```

Comment:
```suggestion
            f"CREATE USER '{self.cluster_admin_user}'@'%' IDENTIFIED BY '{self.cluster_admin_password}';",
```
I think we should have one cluster_admin_user per cluster, not one per instance

Reply:
@delgod , cluster admin commands should only be run from the operator context, so that's why I've added it in per instance user.
If we do otherwise, one could run cluster admin commands even from client applications. Is that ok?

Reply:
Refactor with [ref](https://docs.oracle.com/cd/E17952_01/mysql-shell-8.0-en/innodb-cluster-user-accounts.html) in mind.

Reply:
Will do on new PR